be/src/env/env_util.h

```diff
@@ -30,14 +31,21 @@ struct WritableFileOptions;
 
 namespace env_util {
 
-Status open_file_for_write(Env *env, const std::string& path, std::shared_ptr<WritableFile> *file);
+Status open_file_for_write(Env* env, const std::string& path, std::shared_ptr<WritableFile>* file);
 
-Status open_file_for_write(const WritableFileOptions& opts, Env *env,
-                           const std::string& path, std::shared_ptr<WritableFile> *file);
+Status open_file_for_write(const WritableFileOptions& opts, Env* env, const std::string& path,
+                           std::shared_ptr<WritableFile>* file);
 
-Status open_file_for_random(Env *env, const std::string& path,
-                            std::shared_ptr<RandomAccessFile> *file);
+Status open_file_for_random(Env* env, const std::string& path,
+                            std::shared_ptr<RandomAccessFile>* file);
+
+// A utility routine: write "data" to the named file.
+extern Status write_string_to_file(Env* env, const Slice& data, const std::string& fname);
+// Like above but also fsyncs the new file.
+extern Status write_string_to_file_sync(Env* env, const Slice& data, const std::string& fname);
```

Comment:
extern here seems useless can can be removed.

Reply:
done